Source/Scene/TileMapServiceImageryProvider.js

```diff
@@ -97,30 +99,14 @@ define([
         }
         //>>includeEnd('debug');
 
-        var url = options.url;
-
+        var url = appendForwardSlash(options.url);
         this._url = url;
-        this._ready = false;
```

Comment:
For performance, we should still initialize all these properties in the constructor to `undefined` so we don't risk the object going into dictionary mode when we add the properties in `reinitialize`.